src/formats/pilot.h

```diff
@@ -41,9 +41,9 @@ typedef struct {
     uint16_t offense;        ///< Offense preference value (100 high, should be under 200).
     uint16_t defense;        ///< Defense preference value (100 high, should be under 200).
     int32_t money;           ///< Amount of money the pilot currently has
-    uint8_t color_1;         ///< Color 1 field for the HAR (0-15). 255 means random.
-    uint8_t color_2;         ///< Color 2 field for the HAR (0-15). 255 means random.
-    uint8_t color_3;         ///< Color 3 field for the HAR (0-15). 255 means random.
+    uint8_t color_1;         ///< Color 1 field for the HAR (0-15). 255 means random. 16 is default in tournament.
```

Comment:
by "default" do you mean it came from the PIC?

Reply:
when creating a tournament character, their color_1-3 is 16, and yes the color comes from the PIC

Reply:
```c
src\formats\pilot.h
105:    vga_palette palette;  ///< Palette for photo ?
```
should probably update this comment too.
```
//< Palette for photo and HAR (if not using altpals)
```

Reply:
so 0-16 are ALTPAL colors, 16 is color from the PIC and 255 is random?

Reply:
0-15 are definitely the ALTPAL colors.
It might be the case that 16-255 is "use whatever is in the palette field", and the palette field as you mention is initialized from the PIC.

Just checked the git blame, and the "255 means random" comment comes from katajakasa in 2014:
commit 722ca232a162bf7aa20e2388b4154afd710c6f92
I don't think 255=random was ever implemented, unless we consider the colors from the PIC photo "random".